### DISCUSSION TITLE
[Operator] Add support for Trivy Client Parameters for Trivy offline scanning

### DISCUSSION THREAD

**Discussion by deegee-dev** [aquasecurity/starboard#613] [Questions & Help] [answered]
Is it possible to pass the trivy parameter
> --skip-update

to allow offline scanning of vulnerabilities. Assuming I have an custom image pre-loaded with the CVE database in my own private repository accessible to the Openshift/K8 cluster. I am planning to run starboard in an air-gapped environment  

I am aware of #541 but I'm not sure if the parameters included in that enhancement would cover this. 
Thank you for any advise :)

**Comment by danielpacak** [comment, marked as answer]
We do not officially support Starboard run with Trivy plugin in air-gapped environments. The --skip-update flag is just part of a possible solution.

To properly design and support such RFE we've created #617 Feel free to elaborate and share your ideas in the comments of #617 how this might work in a generic way so we can hopefully make it part of Starboard.
